scripts/retrigger_file_validation/usage.md

```diff
@@ -0,0 +1,15 @@
+# How to revalidate failed file validation in Ingest
+
+1. Get the JWT Token from Ingest UI and place it into the `trigger_file_validation` script replacing the value of the `TOKEN` variable
+2. Get the ID of the `submissionEnvelope` you would like to work with and place it into the `trigger_file_validation` script replacing the `id` (`replace_with_id`) in the `SUBMISSION_URL` variable
+3. Execute the `get_invalid_file_metadata.py` script to create a local file that contains file metadata belongs to the files that failed file validation.
+4. Execute `trigger_file_validation.py` script to re-trigger validation of the above files
+
+   This script is doing the following:
+   1. Open the list of file resources to be revalidated
```

Comment:
do we need this? there were already comments in the code

Reply:
IMHO Not everybody checks the code

Reply:
We need this to be in a single place, otherwise the multiple locations get out of sync.
I suggest to put it in the code and add a link from here.

Reply:
I am going to do the opposite and remove the comments from the code. I think it is better to have documentation in the readme and readable code in the code.